customer_satisfaction/streamlit_app.py

```diff
@@ -0,0 +1,96 @@
+import streamlit as st
+import pandas as pd 
+from PIL import Image
+import pickle 
+
+from inference.predict import predict 
+
+def main(): 
+    st.title("End to End Customer Satisfaction Pipeline with ZenML")
+
+
+    high_levbel_image = Image.open('_assets/high_level_overview.png')
+    st.image(high_levbel_image, caption='High Level Pipeline')
+
+    whole_pipeline_image = Image.open('_assets/training_and_deployment_pipeline.png')
+    
+    st.markdown(""" 
+    #### Problem Statement 
+    The objective here is to predict the customer satisfaction score for a given order based on features like order status, price, payment, etc. I will be using the [ZenML](https://zenml.io/) framework to build a production-ready pipeline to predict the customer satisfaction score for the next order or purchase.
```

Comment:
Use the corrections from above in the README to fix this text.

Reply:
Sure